content/rwh.js

```diff
@@ -981,6 +976,31 @@ var ReplyWithHeader = {
     }
   },
 
+  DateFmt: function(d_str) {
```

Comment:
@ofa- Please use `camelCase` for variables. It seems func `DateFmt` accepts `date` object not a string.  May be something like `function (date) {`.